homeassistant/components/sabnzbd/const.py

```diff
@@ -0,0 +1,26 @@
+"""Constants for the Sabnzbd component."""
+from datetime import timedelta
+
+DOMAIN = "sabnzbd"
+DATA_SABNZBD = "sabnzbd"
+
+ATTR_SPEED = "speed"
+BASE_URL_FORMAT = "{}://{}:{}/"
+CONFIG_FILE = "sabnzbd.conf"
+DEFAULT_URL = "localhost:8080"
```

Comment:
This is a bad default URL. Most people run Home Assistant on a dedicated device and this probably won't work. Better to just leave it blank.

Reply:
removed the default and marked URL as Required